03:12:58: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry_worker.py | telemetry_worker | 48] Logger initialized
03:12:58: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: ATTITUDE
03:12:58: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: LOCAL_POSITION_NED
03:12:58: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: ATTITUDE
03:12:58: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: LOCAL_POSITION_NED
03:12:58: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: ATTITUDE
03:12:59: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: ATTITUDE
03:12:59: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: LOCAL_POSITION_NED
03:12:59: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: ATTITUDE
03:12:59: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: LOCAL_POSITION_NED
03:12:59: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: ATTITUDE
03:13:00: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: ATTITUDE
03:13:00: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: LOCAL_POSITION_NED
03:13:00: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: ATTITUDE
03:13:00: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: LOCAL_POSITION_NED
03:13:00: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: ATTITUDE
03:13:01: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: ATTITUDE
03:13:01: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: LOCAL_POSITION_NED
03:13:01: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: ATTITUDE
03:13:01: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: LOCAL_POSITION_NED
03:13:01: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: ATTITUDE
03:13:02: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: ATTITUDE
03:13:02: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: LOCAL_POSITION_NED
03:13:02: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: ATTITUDE
03:13:02: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: LOCAL_POSITION_NED
03:13:02: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: ATTITUDE
03:13:03: [ERROR] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 136] Telemetry data could not be received from drone.
03:13:04: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: ATTITUDE
03:13:04: [ERROR] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 136] Telemetry data could not be received from drone.
03:13:05: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: LOCAL_POSITION_NED
03:13:05: [ERROR] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 136] Telemetry data could not be received from drone.
03:13:06: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: ATTITUDE
03:13:06: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: LOCAL_POSITION_NED
03:13:06: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: LOCAL_POSITION_NED
03:13:06: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: ATTITUDE
03:13:06: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: LOCAL_POSITION_NED
03:13:07: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: ATTITUDE
03:13:07: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: LOCAL_POSITION_NED
03:13:07: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: LOCAL_POSITION_NED
03:13:07: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: ATTITUDE
03:13:07: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: LOCAL_POSITION_NED
03:13:08: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: ATTITUDE
03:13:08: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: LOCAL_POSITION_NED
03:13:08: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: LOCAL_POSITION_NED
03:13:08: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: ATTITUDE
03:13:08: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: LOCAL_POSITION_NED
03:13:09: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: ATTITUDE
03:13:09: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: LOCAL_POSITION_NED
03:13:09: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: LOCAL_POSITION_NED
03:13:09: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: ATTITUDE
03:13:09: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: LOCAL_POSITION_NED
03:13:10: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: ATTITUDE
03:13:10: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: LOCAL_POSITION_NED
03:13:10: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: LOCAL_POSITION_NED
03:13:10: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: ATTITUDE
03:13:10: [INFO] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 111] Received: LOCAL_POSITION_NED
03:13:11: [ERROR] [C:\Users\Hanxi\wargbootcamp\autonomy-bootcamp-2025-p2\modules\telemetry\telemetry.py | run | 136] Telemetry data could not be received from drone.
